language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import { ChildProcess, spawn } from 'child_process';
+import { ChildProcess, spawn, spawnSync } from 'child_process';
```

Comment:
these are the integration tests, I believe we usually also have some unit tests that mock the ledger. Maybe worth adding those here as well?

Reply:
I'm not sure how I could test the new method meaningful differently. To me having the integration test made most sense.